packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -147,7 +147,7 @@ For more information see: $supportedVersionsLink''',
 
       buildProgress.complete();
     } catch (_) {
-      return exit(ExitCode.software.code);
+      return throw ProcessExit(ExitCode.software.code);
```

Comment:
The return shouldn't be necessary if we're throwing right?

Reply:
Probably not! I did a full replace, didn't noticed this case. Gonna change